pkg/gotohelm/transpiler.go

```diff
@@ -674,6 +674,10 @@ func (t *Transpiler) transpileExpr(n ast.Expr) Node {
 		case token.AND:
 			// Can't take addresses in templates so just return the variable.
 			return t.transpileExpr(n.X)
+		case token.SUB:
```

Comment:
This change seems unrelated? Bundling multiple commits into a PR is fine by me but I will ask for tests :D